lib/hexpm_web/controllers/login_controller.ex

```diff
@@ -74,6 +76,22 @@ defmodule HexpmWeb.LoginController do
     )
   end
 
+  defp login(conn, %User{id: user_id, tfa: %{tfa_enabled: true, app_enabled: true}},
+         password_breached: breached?
+       ) do
+    conn
+    |> configure_session(renew: true)
+    |> put_session("two_factor_user_id", %{uid: user_id, return: conn.params["return"]})
```

Comment:
should we standardise on TFA/tfa everywhere instead of TFA/tfa/two_factor/TwoFactor? Should be easier to grep. We can still use "two factor" in html, urls, etc if that's easier for the users but I'd keep all code identifiers similar.